appHooks/apphub/pre.js

```diff
@@ -18,11 +18,16 @@ class AppHub extends AppHook {
   async test () {
     const apphub = await this.file('.apphub')
 
-    if (apphub == null) return
+    if (apphub == null) {
+      this.error('exist')
+      return
+    }
 
     let data = {}
     try {
-      data = JSON.parse((new Buffer(apphub, 'base64')).toString())
+      const buffer = new Buffer(apphub, 'base64').toString()
+      if (buffer.indexOf("{") == -1) return
```

Comment:
why not check if the string is empty, rather than checking if it's not "sorta JSON-like"?